02-intermediate-front-end/events-and-callbacks/readme.md

```diff
@@ -33,77 +76,592 @@ button.addEventListener("click", handleClickEvent)
 button.addEventListener("click", handleClickEvent)
 ```
 
-### Callbacks
+To explain why that only fired once, let's look at a simpler object, like a car:
+
+```js
+var car = {
+  make: "Toyota",
+  model: "Echo",
+  driver: "Robin"
+}
+car.driver = "Andy"
+```
+
+If I say `car.driver = "Andy"` that doesn't **add** a new driver; it just **overwrites** the old one.
+
+The browser basically stores elements and events like this:
+
+```js
+var button = {
+  events: {
+    click: {
+
+    },
+    mouseover: {
+
+    },
+    keydown: {
+
+    }
+    // all the other event types...
+  }
+}
+```
+
+When you add a `click` listener called `handleClickEvent` like this...
+
+```js
+var button = document.querySelector("button")
+var handleClickEvent = function(){
+  console.log("I was clicked!")
+}
+button.addEventListener("click", handleClickEvent);
+```
+
+...it's like you're saying...
+
+```js
+var button = document.querySelector("button")
+button.events.click.handleClickEvent = function(){
+  console.log("I was clicked!")
+}
```

Comment:
Yeah, not married to it. Just can't think of a better way of putting it. Maybe just, "there's already a click event called `handleClickEvent`, so it can't have another one."